File: Handbook_of_Monte_Carlo_Methods_Taimre_Kroese/statan/mcint.m
% mcint.m
c = (2*pi)^(3/2);
H = @(x) c*sqrt(abs(sum(x,2)));
N = 10^6; alpha = 0.05;
x = randn(N,3); y = H(x);
mY = mean(y); sY = std(y);
RE = sY/mY/sqrt(N);
z = icdf('norm',1-alpha/2,0,1);
fprintf('Estimate = %g, CI = (%g, %g)\n', ...
		    mY, mY*(1- z*RE), mY*(1 + z*RE))

File: DS_and_ML_Mathematical_and_Statistical_Methods_Kroese_Taimre/Chapter3/mcint.py
""" mcint.py """
import numpy as np
from numpy import pi

c = (2*pi)**(3/2)
H = lambda x: c*np.sqrt(np.abs(np.sum(x,axis=1)))
N = 10**6
z = 1.96
x = np.random.randn(N,3)
y = H(x)
mY = np.mean(y)
sY = np.std(y)
RE = sY/mY/np.sqrt(N)
print('Estimate = {:3.3f}, CI = ({:3.3f},{:3.3f})'.format(
        mY, mY*(1-z*RE), mY*(1+z*RE)))   